portal_recaptcha/views/website_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+
+        <template
+            id="assets_editor_recaptcha"
+            name="GoogleRecaptcha"
+            inherit_id="website.layout"
+        >
+            <xpath expr="//main" position="inside">
+                <script src="https://www.google.com/recaptcha/api.js" />
+            </xpath>
+        </template>
+    </data>
+</odoo>
```

Comment:
`website_view.xml` seems like a bad name for this file.

Reply:
Fixed